sql/telemetry/firefox_kpi_dashboard_v1/query.sql

```diff
@@ -81,7 +81,19 @@ WITH
   SELECT
     *
   FROM
-    `moz-fx-data-derived-datasets.analysis.growth_dashboard_forecasts_current` ),
+    `moz-fx-data-derived-datasets.analysis.growth_dashboard_forecasts_current`
+  -- We patch in nondesktop forecasts that exclude Firefox for Fire TV.
+  WHERE
+    datasource NOT IN ('nondesktop_global', 'nondesktop_tier1')
+  UNION ALL
+  SELECT
+    * EXCEPT (asofdate) REPLACE (REPLACE(datasource, '_nofire', '') AS datasource)
+  FROM
+    `moz-fx-data-derived-datasets.analysis.jmccrosky_test`
```

Comment:
we should copy this table out of analysis and name it something more descriptive than `jmccrosky_test`

Reply:
I agree. And we should probably block merging this query update until that's the case. I will pursue with @jmccrosky.